core/hybrisadaptor.cpp

```diff
@@ -937,6 +1100,77 @@ void HybrisManager::pollEventsCallback(
     manager->pollEvents();
 }
 
+void *HybrisManager::binderEventReaderThread(void *aptr)
+{
+    HybrisManager *manager = static_cast<HybrisManager *>(aptr);
+    static const size_t maxNumEvents = 16;
```

Comment:
Me with my rusty c++: I'd assume that 'static const' forces this to be a stored-in-data value, whereas 'const' would be mere compile time constant.

Reply:
Probably can be changed to non-static. This was taken from the hal thread in the same file. Maybe should combine the threads more, those have quite much in common.